src/server/transports/jsonrpc_transport_handler.ts

```diff
@@ -179,4 +186,24 @@ export class JsonRpcTransportHandler {
         }
         return true;
     }
+
+    private paramsTasksListAreValid(params: ListTasksParams): boolean {
```

Comment:
It looks like transport-agnostic logic. Can this be done in `DefaultRequestHandler` to avoid duplication for other transports?